src/rosdep2/platforms/alpine.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# Copyright (c) 2018, rosdep authors
```

Comment:
'rosdep authors' is not a valid copyright entity. If you want to assign the copyright please use the foundation, Open Source Robotics Foundation

Reply:
I changed it to:
```
# Copyright (c) 2018, Open Source Robotics Foundation, Inc.
# Copyright (c) 2018, SEQSENSE, Inc.
```
Is it OK?

Reply:
OK, thanks for your suggestion.